src/Internal/Types/ScriptLookups.purs

```diff
@@ -1321,29 +1321,30 @@ processConstraint mpsMap osMap = do
           ( scriptHashCredential $ unwrap $ plutusScriptStakeValidatorHash
               plutusScript
           )
+      index <- lift $ addCertificate cert
```

Comment:
We have two styles of indexing in this file, maybe it would be better to follow how it's done for `mint` redeemers and call a `reindexStakeRedeemers` as we do with `reindexMintRedeemers` after building the Redeemer?

Reply:
I'm not sure which is better, however

Reply:
No, we don't need to reindex `Cert` redeemers because we know the exact order of `Cert`s at the time of construction.